geomet/tests/test_geopackage.py

```diff
@@ -19,18 +19,29 @@
 from geomet import geopackage
 
 
-def build_header(magic1=0x47, magic2=0x50, version=0x00, flags=0x01, srid=4326):
+def build_header(
```

Comment:
Looks like all of these changes are just to make sure that the `flake8` part of `tox` passes, right?

Reply:
Yep, that's correct. It turns out, Travis-CI wasn't configured to run style checks, so there was a bit of a backlog.